go.mod

```diff
@@ -522,6 +522,7 @@ require (
 	go4.org/netipx v0.0.0-20230125063823-8449b0a6169f // indirect
 	gomodules.xyz/jsonpatch/v2 v2.4.0 // indirect
 	k8s.io/apiextensions-apiserver v0.31.3 // indirect
+	pgregory.net/rapid v1.1.0 // indirect
```

Comment:
Shouldn't this become a direct dependency now that it is being used?

Reply:
I guess so. Funny that `go mod tidy` didn't fix it.